types/storybook__addon-knobs/index.d.ts

```diff
@@ -47,6 +49,8 @@ export function color(name: string, value: string, groupId?: string): string;
 
 export function object<T>(name: string, value: T, groupId?: string): T;
 
+export function radios(name: string, options: RadioOptions<string>, value: string | null, groupId?: string): string;
```

Comment:
Should the radios be generic, same as the other knob type like select? WDYT?

Reply:
If I correctly understand what you're saying I agree with you. 
Do you mean instead of creating an actual interface of `RadioOptions` to just have it be inline like: `options: { [s: string]: string }`?
Because if so I think that's better, I didn't really have a reason to name and export that interface except the fact that it seemed to follow what some others were doing. I do think that the radio seems to follow the select more closely anyway.
My vote is to change it from what I currently have.

Reply:
yes, I meant to make it more similar to `select<T>` and `array<T>` - I am ambivalent, but uniformity is usually a plus :)

Reply:
btw. do we also need to adapt the `KnobOption` type to cover `radios`?